__tests__/e2e-tests/ForgottenPassword-test.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react-native';
+import { useRouter } from 'expo-router';
+import WelcomeScreens from '@/app/index';
+import { Dimensions } from 'react-native';
+import WelcomeConceptScreen from '@/app/screens/welcome/final_screen';
+
+// Maquetter `useRouter` de `expo-router`
+jest.mock('expo-router', () => ({
```

Comment:
Will this be the same if we change the router? We'll have to change it if the routing mechanism changes